app/scripts/lib/experiment.js

```diff
@@ -157,16 +159,17 @@ define(function (require, exports, module) {
      * Get the experiment group for `experimentName` the user is in.
      *
      * @param {String} experimentName
+     * @param {Object} [additionalInfo] additional info to pass to Able.
```

Comment:
What is the comment convention used in content-server for optional params? Maybe `[additionalInfo={}]`?